lib/nfg_ui/bootstrap/components/modal.rb

```diff
@@ -28,7 +28,11 @@ def render
             content_tag(:div, class: modal_dialog_css_classes, role: 'document') do
               content_tag(:div, class: 'modal-content') do
                 capture do
-                  concat(NfgUi::Bootstrap::Components::ModalHeader.new({ title: title }, view_context).render)
+                  # Optionally, leave title nil
+                  # and the title / dismiss functionality is left out.
+                  if title.present?
+                    concat(NfgUi::Bootstrap::Components::ModalHeader.new({ title: title }, view_context).render)
+                  end
```

Comment:
@farhajawed this would be something we'd want to document, i think... the title is now optional on modal. If left out, the ModalHeader component is left out (also removing dismiss functionality).

Reply:
Added this: https://github.com/network-for-good/nfg_ui_display_app/pull/1/files#diff-2c3e3955d590c66e822b5b4cb68e9138574403814026083b1e0346e60a7a6ccdR189-R206